virt/virtual_machines/virt-installing-qemu-guest-agent.adoc

```diff
@@ -1,6 +1,6 @@
 :_content-type: ASSEMBLY
 [id="virt-installing-qemu-guest-agent"]
-= Installing the QEMU guest agent on virtual machines
+= Installing the QEMU guest agent and VirtIO drivers
 include::_attributes/common-attributes.adoc[]
 :context: virt-installing-qemu-guest-agent
```

Comment:
I think an Additional Resources section for the Win VMs would be a good idea. Link to the [MS device/driver installation documentation](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/). That is likely to be a stable URL.